dataeng/jobs/analytics/SnowflakeMicrobachelorsITK.groovy

```diff
@@ -5,7 +5,7 @@ that queries Snowflake for MicroBachelors learner data and transmits it to edX's
 
 package analytics
 
-import static org.edx.jenkins.dsl.AnalyticsConstants.common_log_rotator
+import static org.edx.jenkins.dsl.Constants.common_logrotator
```

Comment:
I tried seeding with this branch, but got an error:
workspace:/dataeng/jobs/analytics/SnowflakeMicrobachelorsITK.groovy: 8: unable to resolve class org.edx.jenkins.dsl.Constants
 @ line 8, column 1.
   import static org.edx.jenkins.dsl.Constants.common_logrotator

Reply:
Ah, based on your previous comment, you wanted DevopsConstants, not Constants.

Reply:
Thanks. I wasn't sure if there was a good way to test this locally :-O. 

I just pushed the update to (hopefully) fix the import. Thanks for the help testing this out.